gui/src/i18n/config.tsx

```diff
@@ -109,9 +109,7 @@ export const langs = [
 // AppConfig path: https://docs.rs/tauri/1.2.4/tauri/api/path/fn.config_dir.html
 // We doing this only once, don't want an override check to be done on runtime,
 // only on launch :P
-const overrideLangExists = exists(OVERRIDE_FILENAME, {
-  dir: BaseDirectory.AppConfig,
-}).catch(() => false);
```

Comment:
This looks like it changes behaviour, what's this change done for?

Reply:
it doesnt, just that exists doesn't have a dir parameter